tests/io/test_stores.py

```diff
@@ -0,0 +1,237 @@
+#         self.assertFalse(content_non_existent)
+#
+#     def test_read_csv_non_existent(self):
+#         content_non_existent = self.s3_store.read(f"path_non_existent_d1b034.csv")
+#         self.assertFalse(content_non_existent)
+#
+#     def test_read_json(self):
+#         content = self.s3_store.read(f"folder_for_test_dont_delete/metadata.json")
+#         self.assertTrue(content)
+#
+#     def test_read_csv(self):
+#         content = self.s3_store.read(f"folder_for_test_dont_delete/KALUMBURU.csv")
+#         self.assertTrue(content is not None)
+
+class LocalStoreTestCase(TestCase):
+    def setUp(self):
+        with patch('nettle.utils.log_info.LogInfo') as MockClass:
+            log = MockClass.return_value
+        self.local_store = Local(log=log)
+
+    @classmethod
+    def setUpClass(cls):
+        os.makedirs(f"tests/temp_folder_for_test/")
+
+    @classmethod
+    def tearDownClass(cls):
+        os.rmdir(f"tests/temp_folder_for_test/")
+
+    def test_fs(self):
+        self.assertTrue(self.local_store.fs())
+        self.assertTrue(isinstance(self.local_store.fs(), fsspec.implementations.local.LocalFileSystem))
+
+    # def test_list_directory(self):
+    #     list directory not implemented yet in Local
+
+    def test_has_existing_file(self):
+        self.assertTrue(self.local_store.has_existing_file(f"tests/fixtures/metadatas.py"))
+
+    def test_has_existing_file_but_file_dont_exist(self):
+        self.assertFalse(self.local_store.has_existing_file(f"path_non_existent_d1b034"))
+
+    def test_has_existing_file_full_path(self):
+        full_filepath_exists = os.path.join(
+            self.local_store.base_folder,
+            f"tests/fixtures/metadatas.py"
+        )
+        self.assertTrue(self.local_store.has_existing_file_full_path(full_filepath_exists))
+
+    def test_has_existing_file_full_path_but_file_dont_exist(self):
+        full_filepath_dont_exist = os.path.join(
+            self.local_store.base_folder,
+            f"path_non_existent_d1b034"
+        )
+        self.assertFalse(self.local_store.has_existing_file_full_path(full_filepath_dont_exist))
+
+    def test_write_none_json(self):
+        with self.assertRaises(Exception) as cm:
+            self.local_store.write(f"tests/temp_folder_for_test/test_metadata_none.json", None)
+        self.assertEqual(
+            "[store.write] content file not identified",
+            str(cm.exception)
+        )
+        os.remove(f"tests/temp_folder_for_test/test_metadata_none.json")
+
+    def test_write_none_csv(self):
+        with self.assertRaises(Exception) as cm:
+            self.local_store.write(f"tests/temp_folder_for_test/test_metadata_none.csv", None)
+        self.assertEqual(
+            "[store.write] content file not identified",
+            str(cm.exception)
+        )
+        os.remove(f"tests/temp_folder_for_test/test_metadata_none.csv")
+
+    def test_write_without_termination(self):

```

Comment:
I'm checking if the filepath is passed without a termination file for the filepath parameter (e.g: if instead of metadata.json the user pass only metadata) generates any kind of problem. In the future someone could change the write method and introduce an unwanted change in this behaviour